modules/rhel-removing-rhcos.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-removing-rhcos-{context}"]
+= Removing RHCOS compute machines from a cluster
+
+After you add the Red Hat Enterprise Linux (RHEL) compute machines to your
+cluster, you can remove the the {op-system-first} compute machines.
+
+.Prerequisites
+
+* You have added RHEL compute machines to your cluster.
+
+.Procedure
+
+. View the list of machines and record the node names of the {op-system} compute
+machines:
++
+----
+$ oc get nodes -o wide
+----
+
+. For each {op-system} compute machine, delete the node:
+.. Mark the node as unschedulable by running the `oc adm cordon` command:
++
+----
+$ oc adm cordon <node_name> <1>
+----
+<1> Specify the node name of one of the {op-system} compute machines.
+
+.. Drain all the pods from the node:
++
+----
+$ oc adm drain <node_name> <1>
```

Comment:
$ oc adm drain <node_name> --force --delete-local-data --ignore-daemonsets

otherwise, will meet error message like 
error: DaemonSet-managed pods (use --ignore-daemonsets to ignore): tuned-lghjm, dns-default-qtjts, node-ca-zsjmq, machine-config-daemon-wbhm4, node-exporter-d4wlp, multus-29h4d, ovs-8x7pm, sdn-bc4sc; pods with local storage (use --delete-local-data to override): alertmanager-main-1, grafana-dd9d97894-tl6rc, prometheus-k8s-1